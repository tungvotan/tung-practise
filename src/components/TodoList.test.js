import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import TodoList from './TodoList';

const setup = (setupProps = {}) => {
  const wrapper = shallow(<TodoList todos={[
    {
      text: 'Test AddTodo',
      completed: false,
      id: 0
    },
    {
      text: 'Test AddTodo',
      completed: true,
      id: 1
    }
  ]}/>);

  return {
    wrapper
  };
};

describe('App component', () => {
  test('renders without crashing', () => {
    const { wrapper } = setup();
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
