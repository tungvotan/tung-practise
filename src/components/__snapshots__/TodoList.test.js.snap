// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders without crashing 1`] = `
<ul>
  <Todo
    completed={false}
    id={0}
    key="0"
    onClick={[Function]}
    text="Test AddTodo"
  />
  <Todo
    completed={true}
    id={1}
    key="1"
    onClick={[Function]}
    text="Test AddTodo"
  />
</ul>
`;
